docs/source/task-launch.rst

```diff
@@ -84,71 +105,131 @@ We can submit tasks from other tasks.  This allows us to make decisions while
 on worker nodes.
 
 To submit new tasks from a worker that worker must first create a new client
-object that connects to the scheduler.  There is a convenience function to do
-this for you so that you don't have to pass around connection information.
-However you must use this function ``worker_client`` as a context manager to
-ensure proper cleanup on the worker.
+object that connects to the scheduler. There are three options for this:
+
+1. ``dask.delayed`` and ``dask.compute``
+2. ``get_client`` with ``secede`` and ``rejoin``
+3. ``worker_client``
+
+.. _Scheduling Policies: https://distributed.readthedocs.io/en/latest/scheduling-policies.html
+.. _Scheduling State: https://distributed.readthedocs.io/en/latest/scheduling-state.html
+.. _The compute function: https://dask.pydata.org/en/latest/scheduler-overview.html#the-compute-function
```

Comment:
It's not clear that these do anything here.  You may want to build locally 

```
cd doc
make html
```

Also, when referring to internal documentation you should refer to it relatively.  Search the codebase for `:doc:`